ui/raidboss/data/07-dt/trial/zoraal-ja-ex.ts

```diff
@@ -0,0 +1,57 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+const triggerSet: TriggerSet<RaidbossData> = {
+  id: 'EverkeepExtreme',
+  zoneId: ZoneId.EverkeepExtreme,
+  timelineFile: 'zoraal-ja-ex.txt',
+  triggers: [
+    {
+      id: 'Zoraal Ja Ex Forward Half Right Sword',
+      type: 'StartsUsing',
+      netRegex: { id: '937B', capture: false },
+      alertText: (_data, _matches, output) => output.frontRight!(),
+      outputStrings: {
+        frontRight: {
+          en: 'In front of boss + right side of boss',
```

Comment:
I know trying to describe boss-relative positions is pretty cursed in text triggers, since people tend to think of and process left/right differently.  But I'd suggest a little brevity, e.g.: 'Front + Boss's Right'.  And then people can do whatever they want in their config.